packages/core/css/colors.module.css

```diff
@@ -146,13 +149,13 @@
   --psColorsGradientFlowStop0: #2968b2;
   --psColorsGradientFlowStop100: #27aae1;
 
-  --psColorsPrimaryActionBackground: #0084bd;
+  --psColorsPrimaryActionBackground: #0074ab;
   --psColorsPrimaryActionTextDarkTheme: #2ea0d6;
   --psColorsPrimaryActionTextLightTheme: #0074ab;
 
-  --psColorsStatusError: #d21c09;
-  --psColorsStatusInfo: #0084bd;
-  --psColorsStatusSuccess: #009e52;
+  --psColorsStatusError: #c00f00;
+  --psColorsStatusInfo: #0074ab;
+  --psColorsStatusSuccess: #007c3a;
```

Comment:
Why not, right?  Were these also inaccessible?

Reply:
They aren't, but I'm going to remove these changes from this PR since they aren't related to Button.